docs/articles/blog/2020-7-15-introducing-the-testcafe-jenkins-plugin.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+title: "Introducing the TestCafe Jenkins plugin"
+permalink: /media/team-blog/:title.html
+isTeamBlog: true
+author: Pavel Redrukhin, Eugene Titerman
+---
+# Introducing the TestCafe Jenkins plugin
+
+TestCafe offers a number of debugging features, including the capability to automatically capture videos and screenshots during the testing process. You can record all your tests or only those that failed, generate multiple video files or stitch them together, and even single out specific interactions (see [the documentation](https://devexpress.github.io/testcafe/documentation/guides/advanced-guides/screenshots-and-videos.html) for the full list of available options).
+
+But automating this process was not always easy. For a long time, Jenkins users had to manually match completed tests to their artifacts. The recent introduction of the [TestCafe Jenkins plugin](https://plugins.jenkins.io/testcafe/) changes that. Links to screenshots and videos taken during the test now automatically appear on the Jenkins test results page. You don't even need to keep your testing server running — all the necessary files are stored right inside the Jenkins build folder.
+
+<!--more-->
+
+The test results page should look like this with our plugin enabled:
+
+![The updated test results page](/testcafe/images/jenkins-plugin/test-results-page.png)
+
+## Test case: the disappearing button
+
+Let's say, the 'Add to Cart' button keeps disappearing from the mobile version of your website.
```

Comment:
'Keeps disappearing' kind of implies that we already struggled with it for a while. I think Simple Present is a bit more straightforward here. 

```suggestion
Let's say the 'Add to Cart' button disappears on the mobile version of your website.
```